### DISCUSSION TITLE
difference between StyledProperty and AttachedProperty

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#6186] [Q&A] [answered]
Can someone clarify the difference between Styled and Attached properties?

**Comment by MarchingCube** [comment, marked as answer]
Is this clear enough https://docs.avaloniaui.net/docs/authoring-controls/defining-properties#attached-properties?

**Reply by npolyak** [reply]
Mostly clear, thanks
